src/components/ui/fallback/ErrorComponent.tsx

```
@@ -0,0 +1,27 @@
+export const ErrorComponent = () => {
+  const router = useRouter();
+  return (
+    <div className="flex h-screen w-screen flex-col items-center justify-center">
+      <Sorry />
+      <p className="font-gowun mt-10 text-2xl">요청하신 페이지를 찾을 수 없습니다.</p>
+      <Button
+        variant={'solid'}
+        tone={'primary'}
+        size={'m'}
+        children={'뒤로가기'}
+        className="mt-5"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)